src/api/routes/users.ts

```diff
@@ -47,6 +47,13 @@ router.get('/:userId', (req: Request, res: Response, next: NextFunction) => {
         })
 })
 
+
+router.post('/', (req: Request, res: Response, next: NextFunction) => {
+    var userData=req.body
+    userData['timestamp']=dateFormat(new Date(), "yyyy-mm-dd HH-MM")
+    res.status(200).json(userData)
```

Comment:
@EANimesha Can you please update the PR to verify the data sent with the user schema. `req.body` should not contain any field that is not used by the `user` model.